sdk/servicebus/azure-messaging-servicebus/README.md

```diff
@@ -298,29 +246,9 @@ sender.sendMessage(message);
 
 #### Receive messages from a session
 
-Receivers can fetch messages from a specific session or the first available, unlocked session.
+Receiving messages from sessions is similar to receiving messages from a non session enabled queue or subscription. The difference is in the builder and the class you use.
 
-<!-- embedme ./src/samples/java/com/azure/messaging/servicebus/ReadmeSamples.java#L175-L181 -->
-```java
```

Comment:
I can include the code snippet here for creating the session processor in a separate PR if you don't have the sample for it.

Reply:
Yes please! Lets do that in a follow up PR or if you have it handy, please push to my branch.